tests/db_migrator_input/config_db/non-default-config-expected.json

```diff
@@ -1115,6 +1115,6 @@
         "speed": "50000"
     },
     "VERSIONS|DATABASE": {
-        "VERSION": "version_2_0_1"
+        "VERSION": "version_2_0_0"
```

Comment:
w

Reply:
The logic in `advance_version_for_expected_database` is:
  - to advance the version in the expected database if it matches the latest version defined by its migrator
  - not to do anything otherwise.

For example, for buffer migrator, the latest version is `version_2_0_0` so all the expected files should have this version. However, this was updated by a previous PR by mistake. So we need to correct it here.